cmd/print_diff.go

```diff
@@ -0,0 +1,19 @@
+package cmd
```

Comment:
Seems a bit out of place to have this as a standalone in `cmd` package. Probably not necessary to move it at this point, but I wonder if it might make more sense in a subpackage of `ui` someday. dk probably has opinions on that.